lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2841,14 +2839,13 @@ mapping:
 
       workflow_resource_params_mapper:
         type: str
-        default: config/workflow_resource_mapper_conf.yml
         required: false
         desc: |
           This parameter describes how to map users and workflows to a set of workflow
           resource parameter to present (typically input IDs from workflow_resource_params_file).
           If this this is a function reference it will be passed various inputs (workflow model
           object and user) and it should produce a list of input IDs. If it is a path
-          it is expected to an XML or YAML file describing how to map group names to parameter
+          it is expected to be an XML or YAML file describing how to map group names to parameter
           descriptions (additional types of mappings via these files could be implemented but
           haven't yet - for instance using workflow tags to do the mapping).
```

Comment:
I think this help should mention the that `config/workflow_resource_mapper_conf.yml.sample` is an example of such a config file.

Reply:
Done. Added as "sample default path" to the description.